libr/core/cmd_egg.c

```diff
@@ -193,6 +217,24 @@ static int cmd_egg(void *data, const char *input) {
 		}
 	}
 	break;
+	case 'S': // "gS"
+	{
+		RListIter *iter;
+		char *p;
+		eprintf ("Configuration options\n");
+		r_list_foreach (configList, iter, p) {
+			if (r_egg_option_get (egg, p)) {
+				eprintf ("%s : %s\n", p, r_egg_option_get (egg, p));
+			} else {
+				eprintf ("%s : %s\n", p, "");
+			}
+		}
+		eprintf ("\nTarget options\n");
+		eprintf ("%s : %s\n", "arch", core->anal->cpu);
```

Comment:
no need to "arch", "os", "bits"... you can put that directly in the format string